README.markdown

```diff
@@ -373,6 +373,30 @@ The connection is alive during the Lisp session and will be reused in each HTTP
 
 `retrieve-all` and the query language came from [datafly](https://github.com/fukamachi/datafly) and [SxQL](https://github.com/fukamachi/sxql). See those documentations for more informations.
 
+#### Crane
+
+[Crane](http://eudoxia.me/crane/) is an non-opinionated ORM for Common Lisp. To use it with Caveman2 add it to your package-list and `use` it in src/db.lisp. Follow the setup steps listed on the [Crane](http://eudoxia.me/crane/) website. 
+
+Here is an example of using [Crane](http://eudoxia.me/crane/) in an POST route with Caveman2 (this assumes that you are `use`ing [Crane](http://eudoxia.me/crane/) in the web.lisp source file):
+
+```common-lisp
+(deftable person ()
+    (first-name :type text)
+    (last-name :type text))
+
+(defroute ("/people" :method :POST) (&key _parsed)
+  (create-from-plist 'person (plist-keyword-args _parsed))
+  "Created")
+```
+
+The helper creates a `plist` with keywords formatted in a way that [Crane](http://eudoxia.me/crane/) requires from an `alist`, which is the request params are in.
+
+```common-lisp
+(defun plist-keyword-args (args)
+  (alexandria:flatten (loop for val in args
+                         collect (list (alexandria:make-keyword (string-upcase (car val))) (cdr val)))))
```

Comment:
Thank you for the feedback, do you have an idea of how the same functionality can work without interning keywords?